pkg/controllers/rollout/controller.go

```diff
@@ -224,10 +246,21 @@ func waitForResourcesToCleanUp(allBindings []*fleetv1beta1.ClusterResourceBindin
 	return false, nil
 }
 
-// pickBindingsToRoll go through all bindings associated with a CRP and returns the bindings that are ready to be updated.
-// There could be cases that no bindings are ready to be updated because of the maxSurge/maxUnavailable constraints even if there are out of sync bindings.
-// Thus, it also returns a bool indicating whether there are out of sync bindings to be rolled to differentiate those two cases.
-func pickBindingsToRoll(allBindings []*fleetv1beta1.ClusterResourceBinding, latestResourceSnapshotName string, crp *fleetv1beta1.ClusterResourcePlacement) ([]*fleetv1beta1.ClusterResourceBinding, bool) {
+// toBeUpdatedBinding is the stale binding which should be rolled out.
```

Comment:
This struct is also used for the bindings that cannot be rolled out I believe?

Reply:
Maybe it's just me but the name gets me a little bit confused, esp. considering that the method below `pickBindingsToRoll` returns two array of this struct (one of them includes the actual rollout targets, and the other needs only a status refresh)